### PR TITLE
Adding a new template that builds a Windows Server 2016 image with VSCode installed

### PR DESCRIPTION
**Pull request by EmmanuelTsouris**
This CloudFormation template (and readme) will build a new Windows Server 2016 image with Visual Studio Code installed using the **AWS::ImageBuilder::Image** resource. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CloudFormation/Windows/windows-server-2016-with-vscode/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Windows Server 2016 Image with Visual Studio Code Installed
+
+This is a sample template that demonstrates how to use the EC2 Image Builder CloudFormation resources to build a Windows Server 2016 Amazon Machine Image (AMI) with Visual Studio Code installed.
+
+## How this Stack Works
+
+By default, AWS Services do not have permission to perform actions on your instances. So, first the stack will create an [AWS::IAM::Role]() resource which grants AWS Systems Manager (SSM) and EC2 Image Builder the necessary permissions to build an image. Then, an [AWS::IAM::InstanceProfile]() is created, which passes the instance role to the EC2 instance.
```
Comment (austoonz) [inline CloudFormation/Windows/windows-server-2016-with-vscode/README.md:7]: Missing links to both ```AWS::IAM::Role``` and ```AWS::IAM::InstanceProfile```.

File: CloudFormation/Windows/windows-server-2016-with-vscode/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Next, an [AWS::ImageBuilder::InfrastructureConfiguration](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-infrastructureconfiguration.html) resource is created, and the Instance Profile is specified as one of its parameters. This is what tells EC2 Image Builder to use that profile with the EC2 Instance during the build.
+
+To install Visual Studio Code, we use a custom [AWS::ImageBuilder::Component](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-component.html) with the PowerShell commands that download and install the application. The Component includes a simple validation step, along with a test step. This ensures Visual Studio Code is installed and can be run from the PowerShell command line.
+
+The [AWS::ImageBuilder::ImageRecipe](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-imagerecipe.html) ties together the Windows Server 2016 parent image and the Visual Studio Code component.
+
+Finally, the [AWS::ImageBuilder::Image] represents the image build version.
```
Comment (austoonz) [inline CloudFormation/Windows/windows-server-2016-with-vscode/README.md:15]: Missing the link for ```AWS::ImageBuilder::Image```

File: README.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ This repository contains samples that demonstrate various aspects of the [Amazon
 
 The ```Components``` folder will hold sample Image Builder components. The samples demonstrate how certain features of the component management application work, or how to execute certain workflows, such as invoking ```ansible-playbook``` or ```chef-client``` within a component.
 
+### CloudFormation
+
+The ```CloudFormation``` folder contains sample EC2 Image Builder CloudFormation templates. These samples demonstrate how to deploy EC2 Image Builder infrastructure to AWS accounts using CloudFormation.
+
```
Comment (austoonz) [inline README.md:14]: nit: As there are some missing links above so you'll likely be posting a revision anyway, can we move this above ```Components``` (alphabetical).

File: CloudFormation/Windows/windows-server-2016-with-vscode/README.md (hunk 1 above)
Comment (EmmanuelTsouris) [reply CloudFormation/Windows/windows-server-2016-with-vscode/README.md:7]: fixed

File: CloudFormation/Windows/windows-server-2016-with-vscode/README.md (hunk 2 above)
Comment (EmmanuelTsouris) [reply CloudFormation/Windows/windows-server-2016-with-vscode/README.md:15]: fixed

File: README.md (hunk 3 above)
Comment (EmmanuelTsouris) [reply README.md:14]: done

**Pull request merged**